posts/2023-XX-YY-boa-release-17.md

```diff
@@ -0,0 +1,237 @@
+this might look as a small increase, we now pass **6,008** tests more than previously. In any case, the big changes in
+this release are not related to conformance, but to huge internal enhancements and new APIs that you will be able to use.
+
+You can check the full list of changes [here][changelog], and the full information on conformance [here][conformance].
+
+Moreover, this release marks the beginning of a great sponsorship by [Lit Protocol](https://litprotocol.com/). Lit is
+distributed key management for encryption, signing, and compute. This means we can now start properly remunerating our
+developers, up to 40h per week distributed in the team. If you wish to sponsor Boa, you can do so by donating to out
+[open collective][collective]. You can also check [easy][easy_issues] or [good first issues][first_issues].
+
+Furthermore, we now have a new domain for Boa, [boajs.dev][boajs].
+
+[changelog]: https://github.com/boa-dev/boa/blob/v0.17/CHANGELOG.md
+[conformance]: https://boajs.dev/boa/test262/
+[collective]: https://opencollective.com/boa
+[easy_issues]: https://github.com/boa-dev/boa/issues?q=is%3Aopen+is%3Aissue+label%3AE-Easy
+[first_issues]: https://github.com/boa-dev/boa/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
+[boajs]: https://boajs.dev
+
+## Highlights
+
+### Optimizations
+
+#### Constant folding optimization
+
+Constant folding expression is a powerful compiler optimization technique that significantly enhances the efficiency and performance of compiled programs. This optimization, now incorporated in the latest release, aims to reduce runtime overhead by evaluating constant expressions at compile-time.
+
+With constant folding expression optimization, the compiler analyzes expressions involving constants and replaces them with their computed results. This process allows the compiler to transform arithmetic operations, comparisons, and logical expressions into simplified forms, removing unnecessary runtime computations. By eliminating these computations, the optimized program benefits from reduced execution time and improved overall performance.
+
+#### Object Shapes (Hidden classes)
+
+Hidden classes (in boa called "shapes" to avoid confusion with JavaScript classes) is an optimization technique that aims to improve the performance of dynamically-typed programming languages. This optimization works by analyzing the patterns of object creation and dynamically generating hidden classes to represent similar object structures.
+
+In dynamically-typed languages, objects are usually a bag of properties (key and value pairs) that are modified at runtime, which can lead to inefficiencies in terms of memory usage. Hidden classes optimization addresses this by organizing object structures more efficiently, reducing memory overhead, it also unlocks the posability for other optimizations, such as inline caching, which optimizes property access.
```

Comment:
```suggestion
In dynamically-typed languages, objects are usually a bag of properties (key and value pairs) that are modified at runtime, which can lead to inefficiencies in terms of memory usage. Hidden classes optimization addresses this by organizing object structures more efficiently, reducing memory overhead, it also unlocks the possibility for other optimizations, such as inline caching, which optimizes property access.
```

Also, I think this should first explain what exactly is hidden classes, then explain the upsides of it:
>  Hidden classes is an alternative way to structure objects that <how hidden classes work>. With this, objects are stored more efficiently, <rest of the upsides>.